pact/src/main/java/bio/terra/profile/consumer/SamStatusApiPactTest.java

```diff
@@ -0,0 +1,64 @@
+package bio.terra.profile.consumer;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.PactTestExecutionContext;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit.ConsumerPactTest;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import org.broadinstitute.dsde.workbench.client.sam.ApiException;
+import org.broadinstitute.dsde.workbench.client.sam.api.StatusApi;
+import org.junit.Assert;
+
+import java.io.IOException;
+
+
+/**
+ * The only way I've been able to get this to generate a pact successfully so far,
+ * has been running it directly in my IDE
+ * If I put it in a test folder, it doesn't get detected as containing any tests
+ * In this folder, the issue is that it isn't set up with a 'main' method
+ */
+public class SamStatusApiPactTest extends ConsumerPactTest {
+
+
+  @Pact(consumer = "bpm-consumer", provider = "sam-provider")
```

Comment:
Why does it need this annotation given that there are also the overrides at lines 38 and 43?

Reply:
The consumer is required by the annotation, for some reason.
The provider, it turns out, is not - so I removed it.

There is another definition pattern that's a shorthand for this one, where both are needed (I couldn't get that one to work)